lib/linguist/heuristics.yml

```diff
@@ -440,6 +440,10 @@ disambiguations:
     pattern: '(?i:\^(this|super)\.|^\s*~\w+\s*=\.)'
   - language: Scala
     pattern: '(^\s*import (scala|java)\.|^\s*class\b)'
+- extensions: ['.sol']
```

Comment:
This change currently won't do anything because Linguist doesn't know about the `.sol` file extension. Please see other pull requests trying to add the `.sol` extension for background.

Reply:
Just tried this for myself to see if there was an easier way to add Solidity support than defining an entirely new strategy for it (following e-mail discussion with.

It, uh, actually works perfectly fine... 😂

Reply:
Wait, I may have spoken too soon. Linguist identifies a solitary `.sol` file, but acts strangely when there are several. Specifically, only the last `.sol` file it's given contributes anything to the repository's language statistics. Every other Solidity file gets ignored.

I'll need to study this closer...